[2025-07-07T07:46:10.050+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dim_policy.run_etl_dim_policy manual__2025-07-07T07:46:08.873391+00:00 [queued]>
[2025-07-07T07:46:10.057+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dim_policy.run_etl_dim_policy manual__2025-07-07T07:46:08.873391+00:00 [queued]>
[2025-07-07T07:46:10.058+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-07T07:46:10.070+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): run_etl_dim_policy> on 2025-07-07 07:46:08.873391+00:00
[2025-07-07T07:46:10.083+0000] {standard_task_runner.py:57} INFO - Started process 629 to run task
[2025-07-07T07:46:10.103+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dim_policy', 'run_etl_dim_policy', 'manual__2025-07-07T07:46:08.873391+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/etl_dim_policy.py', '--cfg-path', '/tmp/tmptkmvt139']
[2025-07-07T07:46:10.107+0000] {standard_task_runner.py:85} INFO - Job 238: Subtask run_etl_dim_policy
[2025-07-07T07:46:10.248+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_dim_policy.run_etl_dim_policy manual__2025-07-07T07:46:08.873391+00:00 [running]> on host 1622b3ff66d0
[2025-07-07T07:46:10.364+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dim_policy' AIRFLOW_CTX_TASK_ID='run_etl_dim_policy' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T07:46:08.873391+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T07:46:08.873391+00:00'
[2025-07-07T07:46:10.366+0000] {logging_mixin.py:150} INFO - Starting ETL process...
[2025-07-07T07:46:10.387+0000] {base.py:73} INFO - Using connection ID '***_insurance_bi' for task execution.
[2025-07-07T07:46:12.103+0000] {logging_mixin.py:150} INFO - Extracted 200000 records
[2025-07-07T07:46:12.202+0000] {logging_mixin.py:150} INFO - Transformed 200000 records
[2025-07-07T07:46:12.214+0000] {base.py:73} INFO - Using connection ID '***_insurance_dw' for task execution.
[2025-07-07T07:46:15.484+0000] {logging_mixin.py:150} INFO - ETL failed: operator does not exist: character varying <> integer
LINE 6:                     product_id != 746663 OR
                                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
[2025-07-07T07:46:15.487+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_dim_policy.py", line 133, in etl_dim_policy
    result = load_dim_policy(transformed_df)
  File "/opt/airflow/dags/etl_dim_policy.py", line 82, in load_dim_policy
    (row['policy_id'], row['customer_id'], row['product_id'], row['tanggal_mulai'], row['tanggal_berakhir'])
psycopg2.errors.UndefinedFunction: operator does not exist: character varying <> integer
LINE 6:                     product_id != 746663 OR
                                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.

[2025-07-07T07:46:15.504+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_dim_policy, task_id=run_etl_dim_policy, execution_date=20250707T074608, start_date=20250707T074610, end_date=20250707T074615
[2025-07-07T07:46:15.523+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 238 for task run_etl_dim_policy (operator does not exist: character varying <> integer
LINE 6:                     product_id != 746663 OR
                                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
; 629)
[2025-07-07T07:46:15.569+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2025-07-07T07:46:15.602+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
